### PR TITLE
fix to issue 829_

### PR DESCRIPTION
**Pull request by AminTorabi-NOAA**
Update to fix issue 829. This PR specify the correct version of numpy that is compatible with the versions of pyarrow and pandas

## Additions

-

## Removals

-

## Changes

-

## Testing

1.

## Notes

-

## Todos

-

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows project standards (link if applicable)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Target Environment support

- [ ] Windows
- [ ] Linux
- [ ] Browser

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

**Review by shorvath-noaa — DISMISSED** [review]

**Comment by shorvath-noaa** [comment]
Addresses #829

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 pip>=22.1.0
 setuptools>=64.0,<69.0
 wheel
-numpy
+numpy>=1.21.0,<2.0.0
```
Comment (aaraney) [inline requirements.txt:4]: Lets go with `numpy~=1.0` instead.

**Review by aaraney — DISMISSED** [review]

File: requirements.txt (hunk 1 above)
Comment (shorvath-noaa) [reply requirements.txt:4]: @AminTorabi-NOAA can you update this?

Comment (AminTorabi-NOAA) [reply requirements.txt:4]: Sure

**Review by shorvath-noaa — APPROVED** [review]

**Review by shorvath-noaa — APPROVED** [review]

**Comment by aaraney** [comment]
:tada:

**Pull request merged**